### PR TITLE
add ex 1 solution

### PR DESCRIPTION
**Pull request by stevelee2497**

### CODE REVIEW

File: exercise1solution/Program.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                if (IsPrme(i))
+                {
+                    primeNumbers.Add(i);
+                }
+            }
+            return primeNumbers;
+        }
+        static void Main(string[] args)
+        {
+            Func<int, List<int>> createList = PrimeNumbers;
```
Comment (voquanghoa) [inline exercise1solution/Program.cs]: No need to have `createList`, just need to call directly `PrimeNumbers(number).ForEach(Console.WriteLine);`

File: exercise1solution/Program.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace exercise1solution
+{
+    class Program
+    {
+        static List<int> PrimeNumbers(int n)
```
Comment (voquanghoa) [inline exercise1solution/Program.cs]: Method name should begin with a `verb`

File: exercise1solution/Program.cs (hunk 1 above)
Comment (stevelee2497) [reply exercise1solution/Program.cs]: done

File: exercise1solution/Program.cs (hunk 2 above)
Comment (stevelee2497) [reply exercise1solution/Program.cs]: done

**Pull request merged**